app/views/userAccount.scala.html

```diff
@@ -12,47 +12,62 @@
     <div class="container">
         <h1 class="header orange-text">Account</h1>
 
-        <div class="row">
-            <div class="col s12 m12 l3">
-                <div class="card-panel">
-                        <p>You are logged in as @username(user).</p><div><a href="/logout"><button class="btn">Log out</button></a></div>
-                </div>
-            </div>
-        </div>
+        <p>You are logged in as <strong>@username(user)</strong>.</p>
 
         <h3 class="header orange-text">Passkeys</h3>
 
-        <div class="row">
-            <div class="col s12 m12 l9">
-                <div class="card-panel">
-                    <p>You can have up to two passkeys to log in to Janus. One should be on-device (e.g. Touch ID) and the other off-device (e.g. security key or authenticator app).
-                    </p>
-                    <div><button id="register-passkey" class="btn" csrf-token="@{CSRF.getToken.get.value}">Register a new passkey</button></div>
-                </div>
-            </div>
-        </div>
+        <p>Passkeys are a secure way to log in without a password. They are stored on your device and are used to authenticate you when you access Janus credentials or the AWS console.</p>
```

Comment:
They can also be off-device

Reply:
fixed in https://github.com/guardian/janus-app/pull/578/commits/7c16bcff38e5ac19ed81f02c2adb09259516c10f